blog/allblogs.html

```diff
@@ -125,11 +125,10 @@ <h5 class="card-title">Petabyte of Observability Data
             </div>
             <div class="row">
                 <div class="card mx-auto">
-                    <a href="./sigscalr-vs-elasticsearch.html">
-                        <img class="card-img-top" src="/assets/blog-2-main-img.png" alt="blog image">
+                    <a href="./siglens-1025x-faster-than-elasticsearch.html">
+                        <img class="card-img-top" style="object-fit: cover;" src="/assets/benchmark.png" alt="blog image">
                         <div class="card-body">
-
-                            <h5 class="card-title">SigScalr vs Elasticsearch: Scaling Observability
+                            <h5 class="card-title">SigLens 1025x faster than Elasticsearch
```

Comment:
can you use this image instead.
![IMG_4072](https://github.com/siglens/siglens.github.io/assets/6307537/dba09694-7829-4742-b362-51c4b60a25c3)

Reply:
removed the old html file and replaced the image.